benchmark/cluster/vgg16/README.md

```diff
@@ -62,6 +62,28 @@ Setting environment variable: `MKL_NUM_THREADS=1`.
 *The performance gap between Fuild and v2 comes from the network interference.*
 
 
+## Test Result (GPU with flowers)
+
+### Hardware Infomation
+
+- GPU: NVIDIA Tesla P40, Driver Version: 375.26
+- CPU: Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz
+- Network: Mellanox Technologies MT27700 Family [ConnectX-4] (100Gb)
+
+### Accelerate Rate
+
+- Pserver Count: 4
+- Batch Size: 20
+- Metrics: samples / sec
+
+| Trainer Count | local | 4 | 10 | 20 |
+| -- | -- | -- | -- | -- |
+| PaddlePaddle Fluid | 42.49 | 45.32 | - | - |
+| PaddlePaddle v2 (1 port) | 22.06 | 6.0 | - | - |
+
+NOTE: GPU cluster training have bottle neck when doing communications.
+NOTE: increasing port number for PaddlePaddle v2 will not increase the total throughput.
```

Comment:
It's a little hard to reason why increase the number of port could increase the throughput? I thought the throughput is limited by the hardware layer, not the TCP protocol layer.

Reply:
Do you use multi-threads on multiple CPU? @typhoonzero